frontend/src/App.vue

```diff
@@ -31,23 +36,27 @@
             @generatorSelected="onGeneratorSelected"
             @stepvalidated="onStepValidated"
           />
-          <div class="navigation" v-if="prompts.length > 0 && !isDone">
-            <b-button class="btn" :disabled="!stepValidated" @click="next">Next</b-button>
-          </div>
+        </v-col>
+          <v-col v-if="prompts.length > 0 && !isDone" class="bottom-right-col" style="height: 4rem;" offset-lg="9" lg="3">
```

Comment:
we are not consistent! here it is is -> v-if="prompts.length > 0

Reply:
Done